pkg/util/hostname.go

```diff
@@ -30,8 +31,16 @@ var (
 		"localhost6.localdomain6",
 		"ip6-localhost",
 	}
+	hostnameExpvars  = expvar.NewMap("hostname")
+	hostnameProvider = expvar.String{}
+	hostnameError    = expvar.String{}
 )
 
+func initHostnameExpvars() {
```

Comment:
This should be done only once, usually in the `init` func.

Reply:
Oh okay I didn't knew the init function was called automatically in go